its/ruling/src/test/resources/sources/azureresourcemanager/rules/S6385.bicep

```diff
@@ -0,0 +1,33 @@
+resource noncompliant 'Microsoft.Authorization/roleDefinitions@2022-04-01' = {
+  name: 'Issue: any action is allowed with sensitive assignable scope and subscription.id()'
+  properties: {
+    permissions: [
+      {
+        actions: [
+          '*'
+        ]
+        notActions: []
+      }
+    ]
+    assignableScopes: [
+      subscription().id
+    ]
+  }
+}
+
+resource compliant 'Microsoft.Authorization/roleDefinitions@2022-04-01' = {
+  name: 'Compliant - specific actions are specified'
```

Comment:
I was thinking - maybe for ITs we should place compliant example before non-compliant? Since we are not checking issues in detail and only the first line, it would make sense to verify this way that we are not triggering on compliant examples.